artiq/devices/novatech409B/controller.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python3
+
+# Copyright (c) 2014 Joe Britton, Sebastien Bourdeauducq
+
+import argparse
+from artiq.protocols.pc_rpc import simple_server_loop
+import importlib
+import novatech409B
+importlib.reload(novatech409B)
```

Comment:
When using an interactive python session reload() is necessary to force reloading of python modules that may have been edited. Python isn't smart enough to automatically reload modified modules.